### ISSUE TITLE
WASM=2 and symbol maps

### ISSUE THREAD

**Issue by crsib** [emscripten-core/emscripten#10593]
Using `WASM=2` along with `--emit-symbol-map` option results in
- Unsrtripped WASM binary
- Not minified asm.js file.

Looks like this happens, because of passing `--emit-symbol-map` sets the `intermediate_debug_info` to true, which is later passed to `shared.Building.wasm2js` as a `debug_info` parameter.

**Comment by kripken** [comment]
Thanks @crsib !

The wasm not being stripped should be fixed in #10633 

By `not minified asm.js file` do you mean the `.wasm.js` file? The compiled code inside it looks minified (which is almost all the file). It is true the non-compiled code is not minified, but I think that's the same with or without symbol files?

**Comment by crsib** [comment]
Hi, @kripken !

Yes, indeed. We worked around it with the following patch, for now, seems to be working fine for us

```
--- source_subfolder/upstream/emscripten/emcc.py	2020-02-27 12:37:41.000000000 +0300
+++ source_subfolder/upstream/emscripten/emcc.py.fixed	2020-02-27 12:38:56.000000000 +0300
@@ -3277,7 +3277,7 @@
                                       opt_level=options.opt_level,
                                       minify_whitespace=optimizer.minify_whitespace,
                                       use_closure_compiler=options.use_closure_compiler,
-                                      debug_info=intermediate_debug_info,
+                                      debug_info=debug_info,
                                       symbols_file=symbols_file)
     if shared.Settings.WASM == 2:
       shutil.copyfile(wasm2js, wasm2js_template)

```

**Comment by kripken** [comment]
@crsib sounds good. does the landed PR also fix it for you? (can test with `emsdk install tot`)
